src/bench/crypto_hash.cpp

```diff
@@ -189,10 +189,12 @@ static void SHA512(benchmark::Bench& bench)
 
 static void SipHash_32b(benchmark::Bench& bench)
 {
-    uint256 x;
-    uint64_t k1 = 0;
+    FastRandomContext rng(true);
+    auto k0 = rng.rand64(), k1 = rng.rand64();
+    auto val = rng.rand256();
     bench.run([&] {
-        *((uint64_t*)x.begin()) = SipHashUint256(0, ++k1, x);
+        ankerl::nanobench::doNotOptimizeAway(SipHashUint256(k0, k1, val));
+        ++k0; ++k1; ++(*val.begin());
```

Comment:
nit: please put multiple statements on multiple lines, this is pretty hard to read code as-is :sweat_smile:

Reply:
Thanks, done